lib/image_optim/config.rb

```diff
@@ -157,6 +157,11 @@ def cache_worker_digests
       !!get!(:cache_worker_digests)
     end
 
+    def timeout
+      timeout = get!(:timeout)
+      timeout ? timeout.to_i : 0
```

Comment:
There is only one place for checking `timeout` value and there it can be `timeout && timeout > 0`. But overall the difference is in only in `timeout: false` or `timeout: nil` been clearer to disable timeout than `timeout: 0`.